vpr/src/route/rr_node.cpp

```diff
@@ -25,7 +25,13 @@ const char* t_rr_node::direction_string() const {
 }
 
 const char* t_rr_node::side_string() const {
```

Comment:
Probably need to deprecate this one too?

Reply:
Seems reasonable to deprecate.

Reply:
@litghost @vaughnbetz The method has been deprecated